Classes/Domain/Repository/User/FrontendUserRepository.php

```diff
@@ -57,21 +58,7 @@ class FrontendUserRepository extends \TYPO3\CMS\Extbase\Domain\Repository\Fronte
 	 * @inject
 	 */
 	protected $typoScriptService = NULL;
```

Comment:
The injected TypoScriptService can be removed at this place, since it will be inherited from the `AbstractRepository`